_includes/devdoc/bitcoin-core/rpcs/rpcs/fundrawtransaction.md

```diff
@@ -64,6 +64,11 @@ All existing inputs must have their previous output transaction be in the wallet
   t: "numeric (bitcoins)"
   p: "Optional<br>(0 or 1)"
   d: "The specific feerate  you are willing to pay(BTC per KB). If not set, the wallet determines the fee"
+  
+- n: "→ <br>`subtractFeeFromOutputs`"
```

Comment:
This field is already documented (see lines 68-71 in the original or 73-76 in this version), and in this call the parameter is an array of integers not a bool.